### ISSUE TITLE
Update-PowerNSX on macOS/Linux

### ISSUE THREAD

**Issue by pandom** [vmware/powernsx#84]
Update PowerNSX cmdlet does not work on macOS.

```
PS /Users/Anthony> update-powernsx -Branch master                                     Exception calling "GetCurrent" with "0" argument(s): "Operation is not supported on
this platform."
At /Users/Anthony/Documents/Github/powernsx/PowerNSX.psm1:3922 char:10
+     if ( -not ( ([Security.Principal.WindowsPrincipal] [Security.Prin ...
+          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : PlatformNotSupportedException

new-object : Cannot find type [Net.WebClient]: verify that the assembly containing
this type is loaded.
At /Users/Anthony/Documents/Github/powernsx/PowerNSX.psm1:3937 char:15
+         $wc = new-object Net.WebClient
+               ~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidType: (:) [New-Object], PSArgumentException
    + FullyQualifiedErrorId : TypeNotFound,Microsoft.PowerShell.Commands.NewObjectCo
   mmand```

**Comment by nmbradford** [comment]
resolved in PR #89 for linux - havent tested on OSX yet, but expect it to work.  Reopen the issue if you have problems.

**Comment by nmbradford** [comment]
oh - you will have to manually run the installer to get the updated update-powernsx :)  chicken and egg and all.  Run the following to invoke the installer (which supports core as well now.

```
$pp = $ProgressPreference;$global:ProgressPreference = "silentlycontinue"; $Branch="master";$url="https://raw.githubusercontent.com/vmware/powernsx/$Branch/PowerNSXInstaller.ps1"; try { try { $response = Invoke-WebRequest -uri $url; $scr = $response.content } catch { if ( $_.exception.innerexception -match "(407)") { $credentials = Get-Credential -Message "Proxy Authentication Required"; $response = Invoke-WebRequest -uri $url -proxyCredential $credentials; $scr = $response.content } else { throw $_ }}; $scr | iex } catch { throw $_ };$global:ProgressPreference = $pp
```
